crates/core/tedge_mapper/src/c8y/converter.rs

```diff
@@ -432,7 +432,12 @@ where
                         .or_insert_with(Operations::default);
 
                     add_or_remove_operation(message, child_op)?;
-                    Ok(Some(create_supported_operations(&message.ops_dir)?))
+                    let path = message.ops_dir.to_path_buf().join(&message.operation_name);
+                    if path.is_dir() {
+                        Ok(Some(create_supported_operations(&path)?))
```

Comment:
I thought we all had agreed to create the device with SmartREST `101` message with the the proper device name, type etc, instead of using empty 114 message.

Reply:
This PR is no more required. Addressed all the issues as part of #1538